### PR TITLE
SEO-3550 WPF changes

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @gowthamanthavasiyappan,
Please find changes made.
Regards,
Scholar.

### CODE REVIEW

File: wpf/DataGrid/Master-Details-View.md [hunk 1]
```diff
@@ -1881,7 +1881,7 @@ this.dataGrid.LiveDataUpdateMode = LiveDataUpdateMode.AllowChildViewUpdate | Liv
 {% endhighlight %}
 {% endtabs %}
 
-You can get the sample from [here](http://www.syncfusion.com/downloads/support/directtrac/general/ze/WPF_-_Sample1341078924).
+You can get the sample from [here](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.DetailsViewDataGrid.html).
```
Comment (gowthamanthavasiyappan) [inline wpf/DataGrid/Master-Details-View.md:1884]: Why you have updated help link instead of FT link?

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: wpf/DataGrid/Master-Details-View.md (hunk 1 above)
Comment (Scholar-jaruha) [reply wpf/DataGrid/Master-Details-View.md]: changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**